doc/src/manual/interfaces.md

```diff
@@ -371,7 +371,7 @@ julia> A[1:2,:]
  2.0  5.0  8.0
 ```
 
-In this example it is accomplished by defining `Base.similar{T}(A::SparseArray, ::Type{T}, dims::Dims)`
+In this example it is accomplished by defining `Base.similar(A::SparseArray, ::Type{S}, dims::Dims)`
```

Comment:
Perhaps keep `T` since that is what is used on L330?
```suggestion
In this example it is accomplished by defining `Base.similar(A::SparseArray, ::Type{T}, dims::Dims)`
```
and perhaps also
```suggestion
In this example it is accomplished by defining `Base.similar(A::SparseArray, ::Type{T}, dims::Dims) where T`
```

Reply:
Yes, good point. I had used `S` because that is used in the table above, but in the code example indeed it is `T`